lib/main/universal_import/html_io.dart

```diff
@@ -1 +1,9 @@
-export 'package:universal_html/src/html.dart';
\ No newline at end of file
+export 'package:universal_html/src/html.dart';
+
+class BroadcastChannel {
+  BroadcastChannel(this.name);
+
+  final String name;
+
+  Stream get onMessage => const Stream.empty();
+}
```

Comment:
why we need it?

Reply:
To be able to build on mobile.

Reply:
Why is BroadcastChannel compiled on mobile? We only allow it to compile on web.

Reply:
Try delete that part and run project on mobile

Reply:
Why did we use it before without any problems? Don't answer like that. Be careful.

Reply:
I wasn't aware of that. Removed